CMakeLists.txt

```diff
@@ -109,42 +103,8 @@ if(BUILD_BENCHMARKS)
   add_subdirectory(benchmarks)
 endif(BUILD_BENCHMARKS)
 
-###################################################################################################
-# - library targets -------------------------------------------------------------------------------
-
-add_library(rmm SHARED "${CNMEM_SOURCE_DIR}/cnmem.cpp")
-
-# override rpath for rmm
-set_target_properties(rmm PROPERTIES BUILD_RPATH "\$ORIGIN")
-
-target_include_directories(rmm PUBLIC
-  "${THRUST_INCLUDE_DIR}"
-  "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
-  "$<INSTALL_INTERFACE:include>"
-  )
```

Comment:
This should also remain, since it tells users of `rmm` where its includes are. You could change `PUBLIC` to `INTERFACE`, shouldn't make a difference either way.

Reply:
Thanks. I tried adding this back, and was hoping this would allow me to remove some of the include_directories from the tests and benchmarks CMakeLists.txt, but it doesn't seem to work. You say it "tells users of `rmm` where its includes are", but how do I make the tests/benchmarks take advantage of this?

Reply:
Done.